python/batch-predictor/setup.py

```diff
@@ -21,11 +21,12 @@
 with open('requirements.txt') as f:
     REQUIRE = f.read().splitlines()
 
-merlin_path = os.path.join(os.getcwd(), "../sdk")
-merlin_sdk_package = "merlin-sdk"
-for index, item in enumerate(REQUIRE):
-    if merlin_sdk_package in item:
-        REQUIRE[index] = f"{merlin_sdk_package} @ file://localhost/{merlin_path}#egg={merlin_sdk_package}"
+# TODO: Uncomment below lines after Pyfunc server stops supporting Python 3.7
+# merlin_path = os.path.join(os.getcwd(), "../sdk")
+# merlin_sdk_package = "merlin-sdk"
+# for index, item in enumerate(REQUIRE):
+#     if merlin_sdk_package in item:
+#         REQUIRE[index] = f"{merlin_sdk_package} @ file://localhost/{merlin_path}#egg={merlin_sdk_package}"
```

Comment:
If my understanding is correct, you commented this block because batch predictor is using 3.7 but merlin-sdk requires >3.7 hence if we keep this block the installation will fail.

Reply:
That is correct. For now, I've pinned the Pyfunc server and batch predictor dependency on `merlin-sdk` to `0.33.0`, which is the last version that's supporting Python 3.7.